testing/test_mark.py

```diff
@@ -477,6 +516,19 @@ def test_some(request):
         reprec = testdir.inline_run("-m", "mark1")
         reprec.assertoutcome(passed=1)
 
+    def assert_markers(self, items, **expected):
```

Comment:
Let's keep it in mind then, and move if necessary. I will make a note of that in the docstring.